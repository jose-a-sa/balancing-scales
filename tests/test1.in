#ScaleName,Left,Right
A1,B1,10
B1,B2,5
B2,B3,B4
B3,3,7
B4,2,B5
B5,1,9