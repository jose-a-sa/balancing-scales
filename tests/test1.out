A1,0,149
B1,0,74
B2,24,0
B3,4,0
B4,17,0
B5,8,0
